service/src/test/java/uk/gov/hmcts/reform/fpl/service/DirectionHelperServiceTest.java

```diff
@@ -681,9 +682,10 @@ void shouldSortDirectionsIntoSeparateEntriesInMapWhenManyAssignees() {
                 .addAll(buildDirections(COURT))
                 .build();
 
-            Map<String, List<Element<Direction>>> sortedDirections = service.sortDirectionsByAssignee(directions);
+            Map<DirectionAssignee, List<Element<Direction>>> sortedDirections = service.sortDirectionsByAssignee(
+                directions);
```

Comment:
I don't think this is spaced down correctly

Reply:
will move onto new line after the =

Reply:
never mind - think this is actually spaced correctly
<img width="873" alt="image" src="https://user-images.githubusercontent.com/15530123/69641412-12650c00-1058-11ea-9b9f-7b2c03c4fa72.png">